bqn-mode.el

```diff
@@ -5,9 +5,10 @@
 ;; Changes are copyright 2021 Marshall Lochbaum <mwlochbaum@gmail.com>.
 
 ;; Author: Marshall Lochbaum <mwlochbaum@gmail.com>
-;; Version: 0.0.0
+;; Version: 0.1.0
```

Comment:
Why?

Reply:
This matches the version number from `bqn-mode-pkg.el`